Resources/Prototypes/_Impstation/Traits/quirks.yml

```diff
@@ -0,0 +1,8 @@
+- type: trait
+  id: HeavyweightDrunk
+  name: trait-heavyweightdrunk-name
+  description: trait-heavyweightdrunk-desc
+  category: Quirks
+  components:
+  - type: HeavyweightDrunk
+    boozeStrengthMultiplier: 0.50
```

Comment:
`boozeStrengthMultiplier` is already set to 0.50 in the component, so this is redundant - also, files should end with an empty newline otherwise github hates it and cries